src/test/common/steps/are-you-pregnant.js

```diff
@@ -85,24 +85,24 @@ Then(/^I am informed that I need to select an option for are you pregnant$/, asy
 
 Then(/^I am informed that I need to enter an expected delivery date$/, async function () {
   await assertErrorHeaderTextPresent(pages.areYouPregnant)
-  await assertExpectedDeliveryDateErrorPresent('Enter your baby\'s due date')
+  await assertExpectedDeliveryDateErrorPresent('Enter the due date in the correct format')
 })
 
 Then(/^I am informed that the date is too far in the past$/, async function () {
-  await assertErrorHeaderTextPresent(pages.areYouPregnant, 'The due date you entered is more than 1 month ago')
-  await assertExpectedDeliveryDateErrorPresent('Please call our helpline on 0345 607 6823 to talk about your application.')
+  await assertErrorHeaderTextPresent(pages.areYouPregnant, 'There\'s a problem')
```

Comment:
Shouldn't this be the other type of apostrophe? Same with below too.

Reply:
I thought I'd keep it the same as the other There's a problem headings, then change them all at the same time